docs/on-prem.rst

```diff
@@ -19,6 +19,7 @@ If you prefer non-Docker examples, please go to `confluentinc/examples GitHub re
 
 After you run through the guided tutorial below, apply the concepts you learn here to build your own event streaming pipeline in |ccloud|, a fully managed, cloud-native event streaming platform powered by |ak|. When you sign up for `Confluent Cloud <https://confluent.cloud>`__, use the promo code ``CPDEMO50`` to receive an additional $50 free usage (`details <https://www.confluent.io/confluent-cloud-promo-disclaimer>`__).
 
+.. note:: Instead of running ``cp-demo`` locally with Docker, you can also run it in a cloud IDE using `gitpod.io <https://gitpod.io>`__ by following this `link <https://gitpod.io/#https://github.com/confluentinc/cp-demo>`__
```

Comment:
Can we move this to the `Setup` section instead?  i.e., with verbiage something like "You can run this demo locally with Docker...and do these 3 steps...or you can run this demo in a cloud IDE using `gitpod.io <https://gitpod.io>`__ by following this `link <https://gitpod.io/#https://github.com/confluentinc/cp-demo>`__" (you can wordsmith)

Reply:
Also, sorry if I missed it, but how does a user connect to C3 if it's running in Gitpod?

Reply:
I've moved it to Setup section like this:

![CleanShot 2021-10-06 at 13 58 37](https://user-images.githubusercontent.com/4061923/136197977-a8464fea-c308-4152-b345-5f0181a74f15.jpg)